webapp/channels/src/components/custom_status/expiry_time.tsx

```diff
@@ -55,7 +55,8 @@ const ExpiryTime = ({time, timezone, className, showPrefix, withinBrackets}: Pro
         <>
             <FormattedMessage
                 id='custom_status.expiry.until'
-                defaultMessage='Until'
+                defaultMessage='Until {time}'
+                values={{time: ''}}
```

Comment:
Shouldn't this be instead something like:
```
if (time) {
   <FormattedMessage ... />
}
```
If not we will show something like `Until  ` and nothing after it, right?

Reply:
Looks fine to me:
![image](https://github.com/mattermost/mattermost/assets/52460000/397b6c17-2fde-47b2-a73c-c98a078b160c)

Reply:
I feel this is wrong. The two whitespaces are well handled, but still... they are used in a different way even being the same string. In Spanish I would say slightly different until a time, and until a date.

On top of that, maybe this makes more sense if the value instead of the empty string is the Timestamp value we are using later.

And the withing brackets should also consider language dependent brackets...

Probably we should look deeper into this, but I am going to approve to get the "most glaring issue" be fixed fast.